### PR TITLE
Shorten to minimal code snippets

### PR DESCRIPTION
**Pull request by hanlint**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,102 +101,50 @@ You can use Composer's speedup methods in two ways:
...
-    algorithms=algorithms,
+    model=MNIST_Classifier(num_classes=10),
     train_dataloader=train_dataloader,
-    eval_dataloader=eval_dataloader,
-    optimizers=optimizer,
-    schedulers=CosineAnnealingLR(optimizer, T_max=2),
-    max_duration="2ep")
+    max_duration="2ep",
+    algorithms=[
+        LabelSmoothing(smoothing=0.1),
```
Comment (hanlint) [inline README.md:143]:
```suggestion
        LabelSmoothing(smoothing=0.1),
        BlurPool(),
```

File: README.md [hunk 2]
```diff
@@ -101,102 +101,50 @@ You can use Composer's speedup methods in two ways:
...
-from torch.optim.lr_scheduler import CosineAnnealingLR
+ifrom torchvision import datasets, transforms
 from torch.utils.data import DataLoader
-from torchvision import datasets, transforms
 
-from composer import Trainer, models
-from composer.algorithms import BlurPool, LabelSmoothing
+from composer import Trainer
+from composer.models import MNIST_Classifier
+from composer.algorithms import LabelSmoothing, CutMix, ChannelsLast
```
Comment (hanlint) [inline README.md:132]:
```suggestion
from composer.algorithms import BlurPool, CutMix, ChannelsLast, LabelSmoothing
```

File: README.md [hunk 3]
```diff
@@ -101,102 +101,51 @@ You can use Composer's speedup methods in two ways:
...
-    optimizers=optimizer,
-    schedulers=CosineAnnealingLR(optimizer, T_max=2),
-    max_duration="2ep")
+    max_duration="2ep",
+    algorithms=[
+        LabelSmoothing(smoothing=0.1),
+        BlurPool(),
+        CutMix(num_classes=10),
+        ChannelsLast(),
+        ]
```
Comment (ravi-mosaicml) [inline README.md:147]:
```suggestion
    ]
```

**Comment by moinnadeem** [comment]
image
wanted to show the difference -- this LGTM!

File: README.md [hunk 4]
```diff
@@ -101,102 +101,49 @@ You can use Composer's speedup methods in two ways:
 
 #### Example: Functional API [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mosaicml/composer/blob/dev/notebooks/Composer_Functional.ipynb)
 
-For users who wish to integrate Composer's speedup methods into their existing training code, we provide stateless functional implementations of these methods.
+Integrate our speed-up methods into your training loop with just a few lines of code, and see the results. Here we easily apply [BlurPool](https://docs.mosaicml.com/en/latest/method_cards/blurpool.html) and SqueezeExcite:
```
Comment (moinnadeem) [inline README.md:104]:
```
https://docs.mosaicml.com/en/stable/method_cards/blurpool.html
```

Comment (moinnadeem) [inline README.md:104]:
```suggestion
Integrate our speed-up methods into your training loop with just a few lines of code, and see the results. Here we easily apply [BlurPool](https://docs.mosaicml.com/en/stable/method_cards/blurpool.html) and SqueezeExcite:
```

File: README.md [hunk 5]
```diff
@@ -101,102 +101,49 @@ You can use Composer's speedup methods in two ways:
...
-    eval_dataloader=eval_dataloader,
-    optimizers=optimizer,
-    schedulers=CosineAnnealingLR(optimizer, T_max=2),
-    max_duration="2ep")
+from composer import Trainer
+from composer.algorithms import BlurPool, ChannelsLast, CutMix, LabelSmoothing
+from composer.models import MNIST_Classifier
+
+transform = transforms.Compose([transforms.ToTensor()])
+dataset = datasets.MNIST("data", train=True, download=True, transform=transform)
```
Comment (moinnadeem) [inline README.md:135]:
```suggestion
dataset = datasets.MNIST("data", download=True, transform=transform)
```

File: README.md [hunk 6]
```diff
@@ -101,102 +101,49 @@ You can use Composer's speedup methods in two ways:
...
+
+trainer = Trainer(model=MNIST_Classifier(num_classes=10),
+                  train_dataloader=train_dataloader,
+                  max_duration="2ep",
+                  algorithms=[
+                      LabelSmoothing(smoothing=0.1),
+                      BlurPool(),
+                      CutMix(num_classes=10),
+                      ChannelsLast(),
+                  ])
```
Comment (moinnadeem) [inline README.md:146]:
```suggestion
trainer = Trainer(
    model=MNIST_Classifier(num_classes=10),
    train_dataloader=train_dataloader,
    max_duration="2ep",
    algorithms=[
        LabelSmoothing(smoothing=0.1),
        CutMix(num_classes=10),
        ChannelsLast(),
        ]
)
```

**Review by moinnadeem — CHANGES_REQUESTED** [review]

**Review by moinnadeem — APPROVED** [review]

**Pull request merged**